ethereum/gateway-eth-ts/src/GatewayTsBase.ts

```diff
@@ -134,8 +131,9 @@ export class GatewayTsBase {
     const result = await (tokenId
       ? gatewayToken.verifyTokenByTokenID(owner, tokenId)
       : gatewayToken.verifyToken(owner));
+    console.log(result);
 
-    return result[0];
+    return result[0] as boolean;
```

Comment:
again, should that not be typed at the source. Ok as a tempfix, but we must make sure that we are not overwriting / ignoring null / undefined cases with these.